CRM/Upgrade/Incremental/General.php

```diff
@@ -40,7 +40,7 @@ class CRM_Upgrade_Incremental_General {
    *
    * @see install/index.php
    */
-  const MIN_INSTALL_PHP_VER = '7.1.0';
+  const MIN_INSTALL_PHP_VER = '7.2.0';
```

Comment:
So this is the minimum for install but 7.1 is the minimum for upgrade?

Reply:
This will also make it 7.2 minimum for Upgrade AFAIK

Reply:
I thought the decision was that the last supported 7.1 version would be 5.34 - but if people want more than 1 month then they should use esr

Reply:
I thought it was 5.33 but if its 5.34 then just target the branch at master

Reply:
yeah - my recollection was the discussion was that since we had already cut 5.34 supporting 7.1 that should stand